src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,134 @@
 package core.basesyntax;
 
+import static java.util.Objects.hash;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public static final int DEFAULT_INITIAL_CAPACITY = 16;
+    public static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size;
+    private int threshold;
```

Comment:
redundant field, we need to update it many times, and use (read value) only once - line 26